README.md

```diff
@@ -23,34 +23,41 @@ We only use GPT3.5 because it's fast, highly accurate, cheap, and is available t
 
 ## Features
 
-
 ### 🪄 Create
 
 <img src="https://raw.githubusercontent.com/Welltested-AI/fluttergpt/main/media/create.png" alt="Creating Code using FlutterGPT" width="500"/>
 
-#### Widget from Description
+#### 1. **Widget from Description**
+
 Create flutter widgets based on the description you provide. Be as specific as you like.
 
 `FlutterGPT Create: Widget from Description`
 
-#### Model Class from JSON
+#### 2. **Model Class from JSON**
+
 Create model classes from JSON with null safety in mind. You can also choose to generate Freezed or JsonSerializable modules
 
 command: `FlutterGPT Create: Model Class from JSON`
 
-
-#### Repository Class from Postman Json
+#### 3. **Repository Class from Postman Json**
 
 Convert you postman collection exports json to API repository class
 
 command: `FlutterGPT Create: API Repository from Postman JSON`
 
-#### Complete Code from BluePrint
+#### 4. **Complete Code from BluePrint**
 
 Get complete code from a blueprint of a class or function with the behaviour of functions, state management and architecture of your choice.
 
 command: `FlutterGPT Create: Code from Blueprint`
 
+#### 5. **Create Responsive Widget from Code**
```

Comment:
```suggestion
#### 5. **Create Web and Tablet Counterparts**
```

This might be slightly more self explanatory and avoid confusion between this and the `Create Responsive Widget from description`. We've already renamed the command in package.json.

Reply:
fixed readme